sunkit_image/granule.py

```diff
@@ -0,0 +1,319 @@
+"""
+This module contains functions that will segment images for granule detection.
+"""
+import logging
+
+import numpy as np
+import scipy.ndimage as sndi
+import skimage
+from sklearn.cluster import KMeans
+
+import sunpy
+import sunpy.map
+
+__all__ = [
+    "segment",
+    "get_threshold",
+    "trim_intergranules",
+    "mark_faculae",
+    "kmeans_segment",
+    "cross_correlation",
+]
+
+methods = ["otsu", "li", "isodata", "mean", "minimum", "yen", "triangle"]

```

Comment:
```suggestion
METHODS = ["otsu", "li", "isodata", "mean", "minimum", "yen", "triangle"]
```
This is how constants are labeled in Python, all capitals.
This will need to also be changed in the functions that use this.